src/common/server.ts

```diff
@@ -49,23 +49,40 @@ export type IInitializationOptions = {
 /**
  * Function to execute a command and return the stdout.
  */
-function executeCommand(command: string): Promise<string> {
+function executeCommand(cmd: string, args: string[] = []): Promise<string> {
   return new Promise((resolve, reject) => {
-    exec(command, (error, stdout, _) => {
-      if (error) {
-        reject(error);
+    const child = spawn(cmd, args);
```

Comment:
I think execFile is what we want here https://nodejs.org/api/child_process.html

Reply:
Oh yeah, that looks better.

Reply:
I can change.